.rubocop.yml

```diff
@@ -179,6 +179,12 @@ Style/BlockDelimiters:
 Style/PercentLiteralDelimiters:
   Enabled: true
 
+Style/SafeNavigation:
+  Enabled: true
+  MaxChainLength: 1
+  Exclude:
+    - 'util/automatiek/**/*'
```

Comment:
Why did we exclude `automatiek`? Automatiek used to be a vendored library, but we no longer upstream any patches so it should be fine to normalize it too.

Reply:
I didn't know it.
I excluded it since it seemed like vendored.